src/components/application_manager/src/resumption/resume_ctrl_impl.cc

```diff
@@ -335,6 +347,37 @@ void ResumeCtrlImpl::OnAwake() {
   StartSavePersistentDataTimer();
 }
 
+void ResumeCtrlImpl::SaveLowVoltageTime() {
+  ResetLowVoltageTime();
+  low_voltage_time_ = time(nullptr);
+  LOG4CXX_DEBUG(logger_,
+                "Low Voltage timestamp : " << low_voltage_time_ << " saved");
+}
+
+void ResumeCtrlImpl::ResetLowVoltageTime() {
+  low_voltage_time_ = 0;
+  LOG4CXX_DEBUG(logger_, "Resetting Low Voltage timestamp");
+}
+
+void ResumeCtrlImpl::SaveWakeUpTime() {
+  ResetWakeUpTime();
```

Comment:
why do this extra reset?

Reply:
@yang1070 Right. Removed in bbd93f1cf73de598eb4bbcf1760114547a19e08e